docs/topics/kotlin-language-features-and-proposals.md

```diff
@@ -0,0 +1,384 @@
+[//]: # (title: Kotlin language features and proposals)
+[//]: # (description: Learn about the lifecycle of Kotlin features. 
+The page contains the full list of Kotlin language features and design proposals.)
+
+JetBrains evolves the Kotlin language according to the [Kotlin language evolution principles](kotlin-evolution-principles.md),
+guided by pragmatic design. 
+You can find the list of language feature proposals and their statuses.
+
+## Kotlin language features lifecycle
```

Comment:
My proposal is to put this section somewhere around `docs/kotlin-evolution-principles.html#language-and-tooling-releases`, so that a user won't have to scroll all the way down every time